### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by stephen-gilbert-cko**
- Added "Payment declined" and "Connection timed out" error messages
- Highlight fields for successful payment
- Updated pay button shadow to match theme

### CODE REVIEW

File: src/front-end/index.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
       <div class="input-wrapper card-number">
         <label id="card-label">Card Number</label>
         <div class="card-and-scheme">
-          <div class="card-number-frame"></div>
+          <div id="card-number" class="card-number-frame"></div>
```
Comment (ioan-ghisoi-cko) [inline src/front-end/index.html:41]: You do not need an id. You an do querySelector('.card-number-frame')

File: src/front-end/index.html [hunk 2]
```diff
@@ -51,7 +51,7 @@
         <!-- Expiry Date -->
         <div class="input-wrapper expiry-date">
           <label id="date-label">Expiry Date</label>
-          <div class="expiry-date-frame"></div>
+          <div id="expiration-date" class="expiry-date-frame"></div>
```
Comment (ioan-ghisoi-cko) [inline src/front-end/index.html:54]: You do not need an id. You an do querySelector('.expiry-date-frame')

File: src/front-end/index.html [hunk 3]
```diff
@@ -60,19 +60,24 @@
         <!-- Cvv -->
         <div class="input-wrapper cvv">
           <label id="cvv-label">CVV</label>
-          <div class="cvv-frame"></div>
+          <div id="cvv-code" class="cvv-frame"></div>
```
Comment (ioan-ghisoi-cko) [inline src/front-end/index.html:63]: You do not need an id. You an do querySelector('.cvv-frame')

File: src/front-end/index.html [hunk 4]
```diff
@@ -60,19 +60,24 @@
...
           <div class="hint cvv-hint hide"><span class="line"></span>100</div>
         </div>
       </div>
 
     </div>
 
     <div class="button-area">
-      <button id="pay-button" class="pay" disabled>Pay Now </button>
+      <button id="pay-button" class="pay" disabled>Pay Now</button>
```
Comment (ioan-ghisoi-cko) [inline src/front-end/index.html:71]: You do not need an id. query with the class

File: src/front-end/script.js [hunk 5]
```diff
@@ -2,6 +2,9 @@ const pageLoader = document.querySelector(".loader");
 const form = document.getElementById("payment-form");
 const inputs = document.getElementById("inputs");
 const nameInput = document.getElementById("cardholder");
+const cardInput = document.getElementById("card-number");
```
Comment (ioan-ghisoi-cko) [inline src/front-end/script.js:5]: You do not need an id.

File: src/front-end/script.js [hunk 6]
```diff
@@ -2,6 +2,9 @@ const pageLoader = document.querySelector(".loader");
...
+const dateInput = document.getElementById("expiration-date");
```
Comment (ioan-ghisoi-cko) [inline src/front-end/script.js:6]: You do not need an id.

File: src/front-end/script.js [hunk 7]
```diff
@@ -2,6 +2,9 @@ const pageLoader = document.querySelector(".loader");
...
+const cvvInput = document.getElementById("cvv-code");
```
Comment (ioan-ghisoi-cko) [inline src/front-end/script.js:7]: You do not need an id.

**Comment by stephen-gilbert-cko** [comment]
Code clean up in the latest commits. I updated payButton to use the button class, but can't select the input fields with their classes - possibly due to Frames' inbuilt style logic?

**Comment by stephen-gilbert-cko** [comment]
Requested changes made per my last commit

**Pull request merged**